### PR TITLE
Compound text

### PR DESCRIPTION
**Pull request by angaither**
Adds compound text field.
Contains title & body fields
Saves & displays the data

### CODE REVIEW

File: modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.module [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+/**
+ * Implements hook_field_formatter_view().
+ */
+function dosomething_compoundtext_field_formatter_view($entity_type, $entity, $field, $instance, $langcode, $items, $display) {
+  $element = array();
+  switch ($display['type']) {
+    case 'dosomething_compoundtext_default':
+      foreach ($items as $delta => $item) {
+        print_r($item);
```
Comment (aaronschachter) [inline modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.module]: should this be removed?

File: modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.install [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/**
+ * @file
+ * Install file for the dosomething_compoundtext module.
+ */
+
+/**
+ * Implements hook_field_schema().
+ */
+ function dosomething_compoundtext_field_schema($field) {
+ return array(
```
Comment (aaronschachter) [inline modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.install]: picky, but spacing.

File: modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.module (hunk 1 above)
Comment (angaither) [inline modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.module]: updated!

File: modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.module [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+/**
+ * @file
+ * Provides a custom field compound text.
+ */
+
+/**
+ * Implements hook_field_info().
+ */
+function dosomething_compoundtext_field_info() {
+return array(
```
Comment (aaronschachter) [inline modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.module]: picky, but spacing (again)

**Comment by aaronschachter** [comment]
@angaither This looks great so far.  Can we resolve the last TODO's before we merge in?  That way we can close out #190 and start adding this into the campaign entity.

**Comment by aaronschachter** [comment]
Looks good!  Ship it

**Pull request merged**